src/comm_manager.cpp

```diff
@@ -403,6 +412,49 @@ void CommManager::heartbeat_callback(void)
   this->send_heartbeat();
 }
 
+void CommManager::config_set_callback(uint8_t device, uint8_t configuration)
+{
+  uint8_t requested_device{device};
+  if(device >=Configuration::DEVICE_COUNT)
+    device = Configuration::DEVICE_COUNT;
+  ConfigManager::config_response resp = RF_.config_manager_.attempt_set_configuration(static_cast<device_t>(device), configuration);
+  comm_link_.send_config_status(sysid_, requested_device, resp.successful, resp.reboot_required, resp.message);
+}
+
+void CommManager::config_request_callback(uint8_t device)
+{
+  if(device < Configuration::DEVICE_COUNT)
+    send_config_value(static_cast<device_t>(device));
+  if(device == 0xff)
+    send_all_config_info();
+}
+void CommManager::send_all_config_info()
+{
+  send_config_info_index_ = 0;
+  send_device_info_index_ = static_cast<device_t>(0);
+}
+
+void CommManager::send_device_info(device_t device)
+{
+  uint8_t device_name[20];
+  RF_.board_.get_board_config_manager().get_device_name(device, device_name);
+  uint8_t max_config = RF_.board_.get_board_config_manager().get_max_config(device);
+  comm_link_.send_device_info(sysid_, device, max_config, device_name);
+}
+
+void CommManager::send_config_info(device_t device, hardware_config_t config)
+{
+  uint8_t config_name[20];
+  RF_.board_.get_board_config_manager().get_config_name(device, config, config_name);
+  comm_link_.send_config_info(sysid_, device, config, config_name);
+}
+
+void CommManager::send_config_value(device_t device)
+{
+  uint8_t config = RF_.config_manager_.get_configuration(device);
+  comm_link_.send_config_value(sysid_, device, config);
+}
```

Comment:
What is the purpose of this message type, when it seems that `config_info` already contains this information?

Reply:
`config_info` has the name, and is requested by rosflight_io on connection.
`config_value` is the current config value for the device, which is sent when requested through the `/config_get` service.